zombies.js

```diff
@@ -79,17 +97,115 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+  constructor(name, health, strength, speed){
+    this._pack = [];
+    this._maxHealth = health;
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this.isAlive = true;
+    this.equipped = false;
+  }
+  getPack(){
+    return this._pack;
+  }
+  getMaxHealth(){
+    return this._maxHealth = 100;
+  }
+  checkPack(){
+    console.log(this.getPack());
```

Comment:
Dont just console.log the pack contents. The intent is that you write a custom function that nicely outputs the contents of the pack.